python/src/HSICEstimatorConditionalSensitivity_doc.i.in

```diff
@@ -0,0 +1,18 @@
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::HSICEstimatorConditionalSensitivity
+"Class HSICEstimatorConditionalSensitivity.
+
+Available constructors:
+    HSICEstimatorConditionalSensitivity()
+    HSICEstimatorConditionalSensitivity(covarianceList, X, Y, estimatorType, weightFunction)
+
+Notes
+-----
+This class implements the HSIC estimator for conditional analysis.
```

Comment:
The API doc pages do not appear in the compiled doc because they are not referenced in `statistics_on_sample.rst`.

Reply:
Thank you I update the file.